examples/schema/record-ids.sssom.tsv

```diff
@@ -0,0 +1,10 @@
+# sssom_version: "1.1"
+# curie_map:
+#   HP: http://purl.obolibrary.org/obo/FBbt_
+#   MP: http://purl.obolibrary.org/obo/UBERON_
+#   RI: https://example.org/sets/record-id#
+# mapping_set_id: https://example.org/sets/record-id
+# license: https://creativecommons.org/publicdomain/zero/1.0/
+record_id	subject_id	predicate_id	object_id	mapping_justification
```

Comment:
can you please give more examples of places where the record_id should be the same, and when it should be different? it's not clear to me which fields in the mapping/mapping set should be considered for this

Reply:
I'm still having a hard time understanding what a record is - I added a bit more text in the body of the PR about this